cmd/soroban-cli/src/rpc/mod.rs

```diff
@@ -204,25 +206,42 @@ pub struct SimulateHostFunctionResult {
 
 #[derive(serde::Deserialize, serde::Serialize, Debug)]
 pub struct SimulateTransactionResponse {
-    #[serde(skip_serializing_if = "Option::is_none", default)]
-    pub error: Option<String>,
-    #[serde(rename = "transactionData")]
-    pub transaction_data: String,
-    #[serde(deserialize_with = "deserialize_default_from_null")]
-    pub events: Vec<String>,
     #[serde(
         rename = "minResourceFee",
         deserialize_with = "deserialize_number_from_string"
     )]
-    pub min_resource_fee: u32,
+    pub min_resource_fee: u64,
+    pub cost: Cost,
     #[serde(skip_serializing_if = "Vec::is_empty", default)]
     pub results: Vec<SimulateHostFunctionResult>,
-    pub cost: Cost,
+    #[serde(rename = "transactionData")]
+    pub transaction_data: String,
+    #[serde(deserialize_with = "deserialize_default_from_null")]
+    pub events: Vec<String>,
+    #[serde(
```

Comment:
are there any unit tests that can assert this type of change also?

Reply:
Not yet, I'll add them in a new PR.